qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -86,6 +86,8 @@
             no Windows
             gagent_check_type = fsfreeze
             gagent_fs_test_cmd = "rm -f /tmp/foo; echo foo > /tmp/foo"
+        - check_reset_reboot_shutdown_fsfreeze:
```

Comment:
This patch includes the file changes in old patch 1032, please clear this in this patch.

Reply:
@sitoliu  Thanks, updated.